src/test/scala/io/confluent/examples/streams/IntegrationTestScalaUtils.scala

```diff
@@ -85,25 +86,28 @@ object IntegrationTestScalaUtils {
   def produceValuesSynchronously[V](topic: String, values: Seq[V], driver: TopologyTestDriver)
                                    (implicit valueSerializer: Serializer[V]): Unit = {
     import collection.JavaConverters._
-    IntegrationTestUtils.produceValuesSynchronously(topic, values.asJava, driver, valueSerializer)
```

Comment:
Because we removed those helper, we call the `TopologyTestDriver` method directly -- it's still useful to keep the Scala helpers for explicit Serde support